spec/unit/cookbook_loader_spec.rb

```diff
@@ -187,7 +152,7 @@ def full_paths_for_part(cb, part)
 
     let(:repo_paths) do
       [
-        File.join(CHEF_SPEC_DATA, "kitchen"),
+#        File.join(CHEF_SPEC_DATA, "kitchen"),
```

Comment:
?

Reply:
ah right that was starting off as just a test